cmd/internal/cli/actions.go

```diff
@@ -217,6 +184,33 @@ func replaceURIWithImage(cmd *cobra.Command, args []string) {
 	return
 }
 
+// TODO: Let's stick this in another file so that that CLI is just CLI
+func vmStarter(cmd *cobra.Command, image string, args []string) bool {
```

Comment:
I'm not a huge fan of this design pattern of having a bool function which will sometimes return, but will other times never return.